test/libyul/objectCompiler/jump_tags.yul

```diff
@@ -10,58 +10,42 @@ object "Contract" {
 // optimizationPreset: none
 // ----
 // Assembly:
-//     /* "source":33:54   */
-//   jump(tag_3)
-// tag_1:
-//     /* "source":48:52   */
-//   tag_5
-//     /* "source":50:51   */
-//   0x01
-//     /* "source":48:52   */
+//     /* "source":109:113   */
 //   tag_2
+//     /* "source":111:112   */
+//   0x01
+//     /* "source":109:113   */
+//   tag_1
```

Comment:
Not really sure what aspect of "jump tags" this one is testing and if it still does its job or needs adjustments.

Reply:
I figured it out in the end: https://github.com/ethereum/solidity/pull/14149#discussion_r1194956604